src/resolvers/UserResolver.ts

```diff
@@ -29,6 +29,50 @@ export class UserResolver {
 		return true;
 	}
 
+	@Mutation(() => BooleanResponse)
+	// @UseMiddleware(isAuth)
+	async deleteUser(@Arg('input') input: DeleteUserInput): Promise<BooleanResponse> {
+		if (!(input.firebaseUserUID || input.id))
+			return {
+				errors: [
+					{
+						field: 'input',
+						message: 'cannot identify delete target: need one of id or firebaseUserUID',
+					},
+				],
+			};
+
+		try {
+			const result = await client.query(
+				`DELETE FROM public.users WHERE users.id=$1 OR users."firebaseUserUID"=$2`,
+				[
+					input.id,
+					input.firebaseUserUID,
+				]
+			);
+			console.log(result);
+			if(result.rowCount)
+			{
+				return { success: true };
+			}
+			return { success: false };
```

Comment:
should return an error along with the failed success
`{
 success: false,
 errors: [ {
       field: "id || firebaseUserUID,
       message: "No fields were deleted from the table, please check firebaseUserUID or id"
} ]
}`